tests/tests_ntp_provider1_checkmode.yml

```diff
@@ -0,0 +1,34 @@
+- name: Configure NTP with default provider
```

Comment:
It seems that this file and the idempotency file are almost identical. IMHO it would be better to move the common parts to a file that is then re-used with the different settings. Or use one file that uses the file twice but with different variables (if possible).